react/QtdSelector.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+/**
+* The quantity selector component.
+* It constrols how many products a client wants to buy.
+*/
+class QtdSelector extends Component {
```

Comment:
The component's name is a bit odd. "Qtd" is an abbreviation of the portuguese word "Quantidade". Consider using `QuantitySelector` instead.

Reply:
Done!